app/transcription.py

```
@@ -260,12 +260,19 @@ def check_if_youtube(source: Source):
             ):
                 return Audio(source=source, chapters=chapters)
-            if source.source_file.endswith(("rss", ".xml")):
-                return RSS(source=source)
+            
+            if lower_url.endswith(("rss", ".xml")):
+                # Strong structural signal — attempt RSS parse directly.
+                try:
+                    return RSS(source=source)
+                except Exception:
+                    # feedparser rejected it; fall through to other handlers.
+                    pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)